test/integration/etcd/data.go

```diff
@@ -491,14 +491,22 @@ func GetEtcdStorageDataForNamespace(namespace string) map[schema.GroupVersionRes
 		gvr("node.k8s.io", "v1alpha1", "runtimeclasses"): {
 			Stub:             `{"metadata": {"name": "rc1"}, "spec": {"runtimeHandler": "h1"}}`,
 			ExpectedEtcdPath: "/registry/runtimeclasses/rc1",
-			ExpectedGVK:      gvkP("node.k8s.io", "v1beta1", "RuntimeClass"),
+			ExpectedGVK:      gvkP("node.k8s.io", "v1", "RuntimeClass"),
```

Comment:
/hold

this has to stay at v1beta1 for a release, otherwise a 1.19 API server won't be able to read the data persisted in etcd

Reply:
thank you very much for catching this in PR before it got merged.  Looking again at #74433 that I was used - it's release notes saying that all runtime classes must be re-created. So it makes sense that this PR has to be different. Updated